crates/extensions/c8y_firmware_manager/src/tests.rs

```diff
@@ -272,6 +251,7 @@ async fn handle_request_child_device_with_failed_download() -> Result<(), DynErr
     // Assert EXECUTING SmartREST MQTT message and FAILED SmartREST MQTT message due to missing 'cache' directory.
     mqtt_message_box
         .assert_received([
+            // FIXME: executing status is no more sent when download fails
```

Comment:
You fixed it right?

Reply:
Yes, this has been fixed by https://github.com/thin-edge/thin-edge.io/pull/2600/commits/2f10f938e2ca7098239846a4cd388ee9c7bf1a8f